src/depackers/xz_dec_stream.c

```diff
@@ -567,7 +590,7 @@ static enum xz_ret dec_main(struct xz_dec *s, struct xz_buf *b)
 	 */
 	s->in_start = b->in_pos;
 
-	for ( ;; ) {
+	while (xz_true) {
```

Comment:
??

Reply:
Yeah, I don't know why this was changed either. This is the original.
```C
static enum xz_ret dec_main(struct xz_dec *s, struct xz_buf *b)
{
	enum xz_ret ret;

	/*
	 * Store the start position for the case when we are in the middle
	 * of the Index field.
	 */
	s->in_start = b->in_pos;

	while (true) {
		switch (s->sequence) {
```